docs/design-principles.md

```diff
@@ -1,12 +1,12 @@
 We have collated 8 principles which guide AEA framework development:
 
-* Accessibility: easy to use.
-* Modularity: encourages module creation and sharing.
-* Openness: easily extensible with third party libraries.
-* Conciseness: conceptually simple.
-* Value-driven: drives immediate value for some use case.
-* Low entry barriers: leverages existing programming languages and web protocols.
-* Safety: safe for the user (economically speaking).
-* Goal alignment: seamless facilitation of user preferences and goals.
+* **Accessibility**: easy of use.
+* **Modularity**: encourages module creation and sharing and reuse.
+* **Openness**: easily extensible with third party libraries.
+* **Conciseness**: conceptually simple.
+* **Value-driven**: drives immediate value.
+<!--* **Low entry barriers**: leverages existing programming languages and web protocols.-->
```

Comment:
Please revert and include. This is an important principle.

Reply:
Sorry for not suggesting anything for this originally. 
I thought based on its description, "Low entry barrier" is effectively a combination of "Accessibility" in making things easy, and "Openness" in playing nice with other (existing) libraries.

Reply:
I agree there is overlap but the description points at another issue. Low barriers to transition might be another heading possible. Which is different from accessibility as you could write a very inaccessible library/framework in Python, so you'd have low entry barriers but not accessibility. Similarly, for openness, related but gets at the fact that third party stuff is easily pluggable.